### ISSUE TITLE
Two or more servers

### ISSUE THREAD

**Issue by Heater711** [xenPanda/DCS_Stats#2]
Good morning,
maybe there is a way to get stats from two or more servers simultaneously. Any thoughts ?

**Comment by xenPanda** [comment]
Yes. I believe it is possible, and I do have plans to add the feature but haven't gotten around to it yet.

**Comment by Heater711** [comment]
okay, I'm still waiting ;)

**Comment by xenPanda** [comment]
Multi server capability has been added to the slmod7.6 branch. The branch has many changes so this is probably best as a new db install, but can be done by just making changes to the database as required and copying over the new and updated files. Look at the stats.sql files to see differences or if needed I can send what changes need to be made. This is in development and has only been tested on my dev server at this point...but it does work for me.

**Comment by Heater711** [comment]
> Multi server capability has been added to the slmod7.6 branch. The branch has many changes so this is probably best as a new db install, but can be done by just making changes to the database as required and copying over the new and updated files. Look at the stats.sql files to see differences or if needed I can send what changes need to be made. This is in development and has only been tested on my dev server at this point...but it does work for me.

yeah, please send me the changes need to be made on DB, I will try

**Comment by xenPanda** [comment]
See the Update section of the readme for the slmod7.6 branch. I have included some steps to update without a fresh install.

**Comment by Heater711** [comment]
Hi,
I've setting up two server: Caucaso Server and Syria Server
config.ini is setting up like this for every server:

config.ini for Caucaso: 
[server]
path = C:\\Users\\Administrator\\Saved Games\\caucaso\\Slmod\\SlmodStats.lua
id = Caucaso

config.ini for Syria
[server]
path = C:\\Users\\Administrator\\Saved Games\\syriaprivate\\Slmod\\SlmodStats.lua
id = Syria

I launch the first .bat for Caucaso and I get the stats for caucaso.
When I launch the .bat for Syria the flight hours of Caucaso will be overwritten

**Comment by xenPanda** [comment]
Can you take a screenshot of before and after showing me the which stat(s) is being overwritten and I will check the code.
